src/app/shared/directives/autofocus/directive-autofoocusDiallog.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Directive, ElementRef, OnInit} from '@angular/core';
+
+@Directive({
+  selector: '[kmAutofocusDialog]',
+})
+export class AutofocusDirectiveDialog implements OnInit {
```

Comment:
I'd prefer reusing and slightly adjusting the already existing `kmAutofocus` directive instead of creating almost exact copy of it.